api/cpp/Cargo.toml

```
@@ -57,7 +57,8 @@ raw-window-handle = { version = "0.5", optional = true }
 # Enable image-rs' default features to make all image formats to C++ users
 image = { version = "0.24.0", optional = true }
 
-esp-backtrace = { version = "0.9.0", features = ["panic-handler", "print-uart"], optional = true }
+esp-backtrace = { version = "0.11.0", features = ["panic-handler", "println"], optional = true }
+esp-println = { version = "0.9.0", default-features = false, features = ["uart"], optional = true }
```

If I understand correctly, you depends on esp-println only to enable the uart feature.

I wonder if this works:

```suggestion
esp-backtrace = { version = "0.11.0", features = ["panic-handler", "println", "esp-println/uart"], optional = true }
```

(untested)

---

Unfortunately that doesn't work :(

```
  feature `esp-println/uart` in dependency `esp-backtrace` is not allowed to contain slashes
  If you want to enable features of a transitive dependency, the direct dependency needs to re-export those features from the `[features]` table.
```